firefox_puppeteer/docs/index.rst

```diff
@@ -32,15 +32,10 @@ Libraries
 The following libraries are currently implemented. More will be added in the
 future. Each library is available from an instance of the FirefoxTestCase class.
 
-.. autoclass:: Puppeteer
-   :members:
```

Comment:
It makes it very confusing because some libraries are exposed from the base Puppeteer and other aren't. I don't think this needs to be a part of the user facing documentation.